ui/pages/confirmation/templates/flask/snap-alert/snap-alert.js

```diff
@@ -19,46 +21,7 @@ function getValues(pendingApproval, t, actions) {
           props: {
             snapName,
           },
-          children: [
-            {
-              element: 'Typography',
-              key: 'title',
-              children: title,
-              props: {
-                variant: TYPOGRAPHY.H3,
-                fontWeight: 'bold',
-                boxProps: {
-                  marginBottom: 4,
-                },
-              },
-            },
-            ...(description
-              ? [
-                  {
-                    element: 'Typography',
-                    key: 'subtitle',
-                    children: description,
-                    props: {
-                      variant: TYPOGRAPHY.H6,
-                      boxProps: {
-                        marginBottom: 4,
-                      },
-                    },
-                  },
-                ]
-              : []),
-            ...(textAreaContent
-              ? [
-                  {
-                    element: 'Copyable',
-                    key: 'snap-dialog-content-text',
-                    props: {
-                      text: textAreaContent,
-                    },
-                  },
-                ]
-              : []),
-          ],
+          children: mapToTemplate(content),
```

Comment:
Does this need to be an array?

Reply:
Please leave a comment explaining that we should swap to use `SnapUIRenderer` at some point instead of doing this hack. (applies to all confirmations)